File: designPattern/factory/python/TvProduct.py
from Product import Product

class TvProduct(Product):

	def __init__(self, company, quality):
		self.company = company
		self.quality = quality
		print(company+"회사의 "+quality+"화질로 tv가 만들어졌습니다.")

	def getInfo(self):
		print(self.company + "회사의 "+ self.quality+"화질인 제품입니다.")

File: designPattern/factory/javascript/TvProduct.js
import Product from "./Product";

export default class TvProduct extends Product{

	constructor(compnay, resoluation){
		super();
		this.compnay = compnay;
		this.resoluation = resoluation;
		console.log(`${this.compnay}에서 ${this.resoluation}화질의 제품이 제작되었습니다.`);
	}

	getInfo(){
		console.log(`${this.compnay}에서 만든 ${this.resoluation}화질의 제품입니다.`);
	}

}